pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -39,6 +42,8 @@ def __init__(
         validate_data: bool = True,
         control_columns: Optional[List[str]] = None,
         yearly_seasonality: Optional[int] = None,
+        lagging_function: str = "geometric",
+        saturation_function: str = "logistic",
```

Comment:
I might suggest `adstock` and `saturation` as names. Especially if they are not actually functions

Reply:
Names changes to `adstock_type` and `saturation_type`